Loop/Extensions/Debug.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Debug.swift
+//  Loop
+//
+//  Created by Michael Pangburn on 3/5/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+func assertingDebugOnly(file: StaticString = #file, line: UInt = #line, _ doIt: () -> Void) {
```

Comment:
Isn't this just going to, by default, report the file and line of this function?

Reply:
Those compiler directives are resolved wherever the function is invoked, so the crash will report the call site.

Reply:
Ah, I see. From the docs: "When used as the default value of a function or method parameter, the special literal’s value is determined when the default value expression is evaluated at the call site." Cool! Looks good.